pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -113,16 +114,20 @@ type PipelineTrigger struct {
 
 // PipelineRunStatus defines the observed state of PipelineRun
 type PipelineRunStatus struct {
-	Conditions duckv1alpha1.Conditions `json:"conditions"`
+	duckv1beta1.Status `json:",inline"`
```

Comment:
One thing this includes that I don't see here is `ObservedGeneration`.

For mutable types, this is important to understand whether the overall Status reflects the latest Spec.

Basically, if `metadata.generation == status.observedGeneration`, then I can believe the conditions.

Not for this change, but probably worth an issue.

Reply:
ah I see, yeah we'll need to update our usage of `Condition` to reflect that then :angel: 
(adding this to my todo :upside_down_face: )